operator/cluster/upgrade.go

```diff
@@ -16,524 +16,628 @@ package cluster
 */
 
 import (
-	"encoding/json"
-
-	"strconv"
-	"strings"
-	"time"
+	"errors"
+	"io/ioutil"
 
 	crv1 "github.com/crunchydata/postgres-operator/apis/crunchydata.com/v1"
 	"github.com/crunchydata/postgres-operator/config"
 	"github.com/crunchydata/postgres-operator/events"
 	"github.com/crunchydata/postgres-operator/kubeapi"
 	"github.com/crunchydata/postgres-operator/operator"
 	"github.com/crunchydata/postgres-operator/util"
+
 	log "github.com/sirupsen/logrus"
+
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
+
+	"fmt"
+	"strconv"
+	"time"
```

Comment:
Please keep the system packages group up at the top

Reply:
updated.